src/app/app.component.html

```diff
@@ -115,16 +144,31 @@
           <button class="btn btn-primary" id="dropdown_{{product.name}}" ngbDropdownToggle>Välj Protein</button>
           <div ngbDropdownMenu aria-labelledby="dropdownBasic1">
             <div *ngFor="let ingredient of product.ingredients">
-              <div *ngIf="ingredient.available == true">
-                <button class="dropdown-item" id="{{product.name}}_{{ingredient.name}}"
-                        (click)="addProduct(product.id, product.name, product.price, ingredient.id, ingredient.name, ingredient.price)">
-                  {{ingredient.name}} ({{ingredient.price + product.price}} kr)
-                </button>
+              <div *ngIf="product.name.toUpperCase() == 'Korean tacos'.toUpperCase()">
+                <div *ngIf="ingredient.available == true">
+                  <button class="dropdown-item" id="{{product.name}}_{{ingredient.name}}"
+                          (click)="addProduct(product.id, product.name, product.price, ingredient.id, ingredient.name, 0)">
+                    {{ingredient.name}} ({{product.price}} kr)
+                  </button>
+                </div>
+                <div *ngIf="ingredient.available != true">
+                  <button class="dropdown-item disabled" id="{{product.name}}_{{ingredient.name}}">
+                    {{ingredient.name}} ({{product.price}} kr)
+                  </button>
+                </div>
               </div>
-              <div *ngIf="ingredient.available != true">
-                <button class="dropdown-item disabled" id="{{product.name}}_{{ingredient.name}}">
-                  {{ingredient.name}} ({{ingredient.price + product.price}} kr)
-                </button>
+              <div *ngIf="product.name.toUpperCase() != 'Korean tacos'.toUpperCase()">
```

Comment:
Why are you hardcoding product name here?

Reply:
That dish should not have extra ingredient cost. and we did the first thing we thought of.

Reply:
`ingredient` is an association, right? Perhaps you could check if there are no `ingredient` associated with a Product and only add the possibility to choose ingredients if there is. Beats hard coding the dish name (or names) IMO.

Reply:
there is ingredients associated with this product, and only on products with ingredients you have the possibility to choose ingredient, the problem with this  one, is that it has the same ingredients as other products(dishes) but should not have the extra cost added.

Reply:
and we were told about that it should not have extra cost added today

Reply:
and we will remove this and create more ingredients with no extra cost to remove this hardcoding.